### ISSUE TITLE
Problems about the hyperparameters and the loss

### ISSUE THREAD

**Issue by huihao96** [Turoad/CLRNet#52]
Thank you for the great work!
Just have some problems about the hyperparameters you used: the batch size, the learning rate etc. for each dataset. The default values show in the config files seem different from what you described in the paper. What was the value you used and how was the loss? 
Sometimes the loss is low but I got bad result. Could you give me some advice(image size was modified, different confident threshold was tried)?

Thank you in advance!
